go/base/management/commands/go_manage_conversation.py

```
@@ -82,3 +86,27 @@ def handle_command_archive(self, *apps, **options):
         self.stdout.write("Archiving conversation...\n")
         conversation.archive_conversation()
         self.stdout.write("Conversation archived\n")
+
+    def handle_command_export(self, *apps, **options):
+        conversation = self.get_conversation(options)
+        self.stdout.write(json.dumps(conversation.get_data()))
+
+    def load_file(self, options):
+        if 'file' not in options:
+            raise CommandError('Please specify a file to load.')
+        file_name = options['file']
+        return open(file_name, 'r')
+
+    def handle_command_import(self, *apps, **options):
+        raw_conv_data = json.load(self.load_file(options))
+        allowed_keys = [
+            'conversation_type',
+            'description',
+            'name',
+            'config',
+            ]
+
+        new_conv_data = dict([(key, raw_conv_data[key]) for key in
+                              allowed_keys])
```

I'd prefer to throw an error here if we have unexpected keys rather than silently discarding them.